pyproject.toml

```diff
@@ -1,7 +1,82 @@
-[tool.semantic_release]
-version_variables = [
-    "setup.py:__version__",
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "yandexcloud"
+version = "0.328.0"
+authors = [{name = "Yandex LLC", email = "cloud@support.yandex.ru"}]
+license = {text = "MIT"}
+description = "The Yandex.Cloud official SDK"
+requires-python = ">=3.9"
+classifiers = [
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+dependencies = [
+    "cryptography>=41.0.7,<43",
+    "grpcio>=1.64.0,<2",
+    "protobuf>=4.25.3,<5",
+    "googleapis-common-protos>=1.63.0,<2",
+    "pyjwt>=2.8.0,<3",
+    "requests>=2.32.3,<3",
+    "six>=1.16.0,<2",
+]
+
+[project.readme]
+file = "README.md"
+content-type = "text/markdown"
+
+[project.urls]
+Homepage = "https://github.com/yandex-cloud/python-sdk"
+
+[dependency-groups]
+dev = [
+    "pre-commit>=4.0.1",
+    "python-semantic-release>=v9.8.8",
+    "tox-gh>=1.3.1",
+    "tox>=4.14.2",
+    "yandexcloud"
+]
+format = [
+    "black>=24.4.2",
+    "isort>=5.13.2",
 ]
+genproto = [
+    "grpcio-tools==1.59.3",
```

Comment:
Strictly pinned versions for some packages used in lint and format, because newer versions produce new errors. Will relax these restrictions in next PR, where also will fix linting and formating errors